Sources/DatadogObjc/Global+objc.swift

```diff
@@ -0,0 +1,30 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+import Datadog
+
+@objcMembers
+public class DDGlobal: NSObject {
+    public static var sharedTracer: DatadogObjc.OTTracer = noopTracer {
+        didSet {
+            // We must also set the Swift `Global.tracer`
+            // as it's used internally by auto-instrumentation feature.
+            if let swiftTracer = sharedTracer.dd?.swiftTracer {
+                Global.sharedTracer = swiftTracer
+            }
+        }
+    }
+    public static var rum: DatadogObjc.DDRUMMonitor = noopRUMMonitor {
```

Comment:
if `DatadogObjc` is just a translation layer then we should somehow use the default Swift `RUMMonitor` here.
if the default Swift monitor changes, default Objc monitor should follow the change: this code doesn't follow, it creates Objc's own default value

Reply:
`DatadogObjc.DDRUMMonitor` (Objc) translates its calls to `Datadog.RUMMonitor` (Swift). Both Swift and Objc SDKs need to provide its `Global` namespace which uses no-op implementation of monitor by default.  This is exactly what this code does and how it reads. I don't see how this can be done differently 🤔.

Reply:
both Swift and Objc don't need to provide their own `NoOp`s; Objc can use Swift's `NoOp` without duplication.
plus, Objc doesn't need to decide what default to use (`NoOp` or `FooOp` or etc.?), it should use Swift's default.

you can do this like this:
```swift
static var rum = ObjC.DDRUMMonitor(swiftRUMMonitor: Global.rum)
```
now Objc gets whatever Swift gives by default

Reply:
Sounds good 👍. I removed the no-op RUM monitor and no-op Tracer from `DatadogObjc`. Now they use whatever Swift SDK exposes as `Global.*`.